extensions/wikia/WikiaInYourLang/tests/WikiaInYourLangControllerTest.php

```diff
@@ -35,6 +35,8 @@ public function getDomainsList() {
 			['http://sandbox-s3.naruto.wikia.com', 'naruto.wikia.com'],
 			['http://sandbox-s3.zh.naruto.wikia.com', 'naruto.wikia.com'],
 			['http://sandbox-s3.pt-br.naruto.wikia.com', 'naruto.wikia.com'],
+			['http://sandbox-mercury.pt-br.naruto.wikia.com', 'naruto.wikia.com'],
```

Comment:
To be in sync with what was here before, let's cover all three domain cases (zh, pt-br, no language domain) for each sandbox address you've added.